src/sentry/api/endpoints/organization_monitors.py

```diff
@@ -70,9 +70,6 @@ def get(self, request, organization):
                         )
                     except ValueError:
                         queryset = queryset.none()
-
```

Comment:
You have to expand the diff up to see, this removes a duplicate if/else branch

Reply:
Are you talking to matt here?

Reply:
Yeah or like anyone reading